sample_files/pre-commit-13.0/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
```

Comment:
Should not we be specific on the file extensions here? In case someone has an exotic file format to include in a module for some reason?

Reply:
This defines the default editor configuration, it should be OK as it is.

Reply:
> default editor configuration

Is it? Or is it a catch all glob pattern that would override any configuration set in higher level `.editorconfig`?

Reply:
Is there any difference? :thinking: